src/extra/widgets/anim_timeline/lv_anim_timeline.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file lv_anim_timeline.h
+ *
+ */
+
+#ifndef LV_ANIM_TIMELINE_H
+#define LV_ANIM_TIMELINE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../../lvgl.h"
+#if LV_USE_ANIM_TIMELINE
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/*Data of anim_timeline*/
+typedef struct {
+    uint32_t start_time;
+    lv_obj_t * obj;
```

Comment:
Can it be `void *` to allow animating anything (style properties, any variable, etc), not only objects?

Reply:
Thanks